README.md

```diff
@@ -88,6 +88,19 @@ Starting package install...
 Successfully installed
 ```
 
+### Scoop
+
+[Scoop](https://scoop.sh/) is a Windows command-line installer and manager for portable applications.
+
+To install with Scoop, run the following commands:
+
+```shell
+scoop bucket add extras
+scoop install extras/jpegview-fork
+```
+
+After installation, the configuration file is located at `C:\Users\<your-username>\scoop\persist\JPEGView-fork\JPEGView.ini`.
```

Comment:
You can use `%USERNAME%` variable here:
```suggestion
After installation, the configuration file is located at `C:\Users\%USERNAME%\scoop\persist\JPEGView-fork\JPEGView.ini`.
```

Reply:
Good call out.

I'll even go one step further: how about %HOMEPATH%, which substitutes the `C:\Users\` as well? This would make the text `%HOMEPATH%/scoop/persist/JPEGView-fork/JPEGView.ini`.

Will update with this. Comment back if you disagree.

Reply:
Actually, the most reliable variable that Windows uses is `%UserProfile%` ... you can make it all uppercase too, but often it's stylized that way to make it easier to read.  Can you adjust it to that?

I'll add the badge when I pull this in.  Thanks for adding it to the scoop official!

Reply:
> %UserProfile%

Works for me. Pushed.